requirements/libraries.txt

```diff
@@ -33,7 +33,8 @@ wagtail-flags==5.2.0
 wagtail-inventory==1.6
 wagtail-placeholder-images==0.1.1
 wagtail-sharing==2.7
-wagtail-treemodeladmin==1.5.0
+git+https://github.com/cfpb/wagtail-treemodeladmin@wag3-support#egg=wagtail-treemodeladmin
+# wagtail-treemodeladmin==1.5.0
```

Comment:
```suggestion
wagtail-treemodeladmin==1.6.0
```

Reply:
How about doing this on `main` instead to reduce the size of the Wagtail upgrade effort? Then we can rebase `upgrade/wagtail`.

Reply:
Thanks @willbarton - now that this upgrade is merged into main in https://github.com/cfpb/consumerfinance.gov/pull/7445, I'm going to rebase `upgrade/wagtail`.

Reply:
Done, and rebased this PR branch against that.